R/zchunk_L102.water.supply.unlimited.R

```diff
@@ -14,7 +14,7 @@
 #' @importFrom tidyr gather spread
 #' @author YourInitials CurrentMonthName 2017
 #' @export
```

Comment:
Remove `@export` tag line (you'll need to rebuild documentation after addressing these two comments)